### ISSUE TITLE
"Enable notifications background" & ambient display 

### ISSUE THREAD

**Issue by brnhrd** [wasdennnoch/AndroidN-ify#1011]
"Enable notifications background" setting does not work properly with ambient display. When a notification arrives on the lockscreen and it starts pulsing, the background of the notification is white and visible, thus making the text unreadable. 
Nexus6P

**Comment by tushar260** [comment]
Person using a module to get Nougat features on a device which officially has Nougat :c

**Comment by brnhrd** [comment]
correct, reason is simple: there is no xposed for nougat yet and i don't wanna miss my other xposed stuff.

**Comment by wasdennnoch** [comment]
Can you please post a screenshot / screenrecord?

**Comment by brnhrd** [comment]
I recorded two videos: 
one with "Enable notifications background" on and one with the setting off. When switched to off, notfications show up as expected when ambient display is pulsing.
https://goo.gl/photos/pUZtmT9hkYw3YSEU6
